LayoutTests/platform/mac-wk2/TestExpectations

```diff
@@ -1974,8 +1974,6 @@ webkit.org/b/268406 transitions/flex-transitions.html [ Pass Failure ]
 imported/w3c/web-platform-tests/webrtc/simulcast/setParameters-active.https.html [ Pass Failure ]
 imported/w3c/web-platform-tests/webrtc/simulcast/h264.https.html [ Pass Failure ]
 
-webkit.org/b/268494 [ Monterey+ Release ] media/track/media-element-enqueue-event-crash.html [ Pass Crash ]
```

Comment:
this shouldn't be removed for now.
as this change doesn't fix the intermittent crash

Reply:
Undid. Although the intention was to update and let this run and see whether the expectation is now needed or not.